php/razorpay_js/index.html

```diff
@@ -0,0 +1,402 @@
+			}
+		}
+
+		async function openRPCheckout(razorpayCustomerDetail, orderId, orderCost, currency) {
+			let options = {
+				key: "YOUR_RAZORPAY_KEY", // Enter the Key ID generated from the Dashboard (eg: rzp_test_*****yoo28mk)
+				amount: orderCost, // Amount is in currency subunits. Default currency is INR. Hence, 50000 refers to 50000 paise
+				currency: currency,
+				name: "Acme Corp",
+				description: "Test Transaction",
+				image: "https://example.com/your_logo",
+				email: "user@email.com",
+				recurring: true,
+				order_id: orderId,
+				handler: async function (response) {
+					if(response.razorpay_payment_id) {
+						let subscriptionDetails = await createSubscrition(
+							response.razorpay_payment_id
+						);
+						alert(`Chargebee Subscription Created: ${subscriptionDetails.subscription.id}`);
+						console.info(
+							`======= Chargebee Subscription Created ========`
+						);
+						console.info(subscriptionDetails);
+						payButton.classList.remove('disable')
+					} else {
+						alert(`Error Creating Chargebee Subscription`);
+						console.info(
+							`======= Error Creating Chargebee Subscription ========`
+						);
+						payButton.classList.remove('disable')
+					}
+				},
+				customer_id: razorpayCustomerDetail.id,
+				prefill: {
+					name: "Test User",
+					email: "test.user@example.com",
+					contact: "9999999999",
+				},
+				notes: {
+					address: "Razorpay Corporate Office",
+				},
+				theme: {
+					color: "#3399cc",
+				},
+			};
+
+			var rzp1 = new Razorpay(options);
+
+			rzp1.open();
+
+			rzp1.on("payment.failed", function (response) {
+				payButton.classList.remove('disable')
+				alert(response.error.code);
+				alert(response.error.description);
+				alert(response.error.source);
+				alert(response.error.step);
+				alert(response.error.reason);
+				alert(response.error.metadata.order_id);
+				alert(response.error.metadata.payment_id);
+			});
+		}
+
+		async function createSubscrition(paymentId) {
+			console.info(
+				`====== Initiating Chargebee Subscription ======`
+			);
+			try {
+				let myHeaders = new Headers();
+				myHeaders.append("Content-Type", "application/x-www-form-urlencoded");
+
+				let urlencoded = new URLSearchParams();
+				urlencoded.append(
+					"subscription_items[item_price_id][0]",
+					ITEM_PRICE_ID
+				);
+				urlencoded.append("subscription_items[billing_cycles][0]", 2);
+				urlencoded.append("subscription_items[quantity][0]", 1);
+				urlencoded.append(
+					"payment_intent[gateway_account_id]",
+					"gw_AzqMB5T4qqHso7xRk"
+				);
+				urlencoded.append("payment_intent[gw_token]", paymentId);
+				urlencoded.append("payment_intent[payment_method_type]", paymentType);
+				urlencoded.append("cb_customer_id", CB_CUSTOMER_ID);
+
+				let requestOptions = {
+					method: "POST",
+					headers: myHeaders,
+					body: urlencoded,
+				};
+
+				let subscriptionDetails = await fetch(
+					"http://localhost:3000/api/create_subscription",
```

Comment:
Extract the server URL out to a global variable, then build the URLs while making the request